sdk/provisioning/Azure.Provisioning.AppConfiguration/tests/BasicAppConfigurationTests.cs

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Threading.Tasks;
+using Azure.Core.TestFramework;
+using Azure.Provisioning.Expressions;
+using Azure.Provisioning.Tests;
+using NUnit.Framework;
+
+namespace Azure.Provisioning.AppConfiguration.Tests;
+
+public class BasicAppConfigurationTests(bool async)
+    : ProvisioningTestBase(async /*, skipTools: true, skipLiveCalls: true /**/)
+{
+    [RecordedTest]
+    [Description("https://github.com/Azure/azure-quickstart-templates/blob/master/quickstarts/microsoft.appconfiguration/app-configuration-store-ff/main.bicep")]
+    public async Task CreateAppConfigAndFeatureFlag()
+    {
+        await using Trycep test = CreateBicepTest();
+        await test.Define(
+            ctx =>
+            {
+                BicepParameter location = BicepParameter.Create<string>(nameof(location), BicepFunction.GetResourceGroup().Location);
+                location.Description = "Config Store location.";
+
+                BicepParameter featureFlagKey = BicepParameter.Create<string>(nameof(featureFlagKey), "FeatureFlagSample");
+                featureFlagKey.Description = "Specifies the key of the feature flag.";
+
+                AppConfigurationStore configStore =
+                    new(nameof(configStore), AppConfigurationStore.ResourceVersions.V2022_05_01)
```

Comment:
Can we have more defaults? I wish all properties but the ones we absolutelly cannot default had defaults.

Reply:
I don't want to _here_.  All these examples are using everything at the lowest level - generated straight from management plane.  The `StorageResources` class is how I want to default the world.  Those are just a little slow to get right and we don't need them in place to unblock the Aspire folks.